src/Less/index.less

```diff
@@ -71,7 +71,7 @@ body {
 
 .wproject {
 	background: #f2f2f2;
-	color: #3f5fbf;
+	color: #2DABAD;
```

Comment:
Maybe it's just me, but #2DABAD looks like a stark teal colour that doesn't fit in with the rest of interface. What if we removed the color property altogether and let it be the default #333 ?

Reply:
Sounds good to me.